### PR TITLE
note about upgrading appliance

### PR DESCRIPTION
**Pull request by michalskrivanek**
Appliance needs to be upgraded to latest bits during deployment

[Bug fix]

Fixes issue https://bugzilla.redhat.com/show_bug.cgi?id=2159724

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md):

This pull request needs review by: @emarcusRH

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
+1

File: source/documentation/common/install/proc-Deploying_the_Self-Hosted_Engine_Using_the_CLI.adoc [hunk 1]
```diff
@@ -25,6 +25,11 @@ When the `he_pause_host` or `he_pause_before_engine_setup` variables are set to
 ** `after_add_host`
 ** `after_setup`
 
+[IMPORTANT]
```
Comment (emarcusRH) [inline source/documentation/common/install/proc-Deploying_the_Self-Hosted_Engine_Using_the_CLI.adoc:28]: I suggest to move this to the first bullet in the Prerequisites, with the following rewording:
"Upgrade the appliance content to the latest product version before performing `engine-setup`.
To do this manually, pause the deployment using `he_pause_before_engine_setup` and perform a `dnf update`. To do this automatically, apply the `enginevm_before_engine_setup`  hook."

**Review by emarcusRH — CHANGES_REQUESTED** [review]

File: source/documentation/common/install/proc-Deploying_the_Self-Hosted_Engine_Using_the_CLI.adoc (hunk 1 above)
Comment (michalskrivanek) [reply source/documentation/common/install/proc-Deploying_the_Self-Hosted_Engine_Using_the_CLI.adoc:28]: @emarcusRH I can't get the syntax right. can you push a new commit with that change please? or let me know how exactly it should look like so it renders properly.
Thanks!

**Review by emarcusRH — APPROVED** [review]

**Comment by emarcusRH** [comment]
@michalskrivanek I made the changes and a new commit via the GitHub site, but it looks like that new DCO signing check is blocking the final approval.  Trying again from Git CLI on my computer...

**Pull request merged**